README.md

```diff
@@ -4,4 +4,4 @@ This is the issue tracking repository for Accessible Name and Description Comput
 
 You can also compare against [the most recent published version here](https://www.w3.org/TR/accname-1.2/).
 
-To open a PR, please see the [main ARIA repository where the specification lives](https://github.com/w3c/aria/accname). This specification is part of the [ARIA suite](https://www.w3.org/WAI/ARIA/deliverables), and uses the same code structure and build tools as other ARIA suites. General information about editing specifications is in the [main ARIA repository readme](https://github.com/w3c/aria/).
```

Comment:
Just FYI: there was already https://github.com/w3c/core-aam/pull/235 which adjusted the core-aam readme to link to https://github.com/w3c/aria/tree/main/core-aam . I'm just mentioning it in case you would prefer to match that.